1_fetch/src/fetch_data.R

```diff
@@ -0,0 +1,12 @@
+# Load necessary packages
+library(sbtools)
+
+# Function to fetch data
+fetch_data <- function(output_path) {
+  # define file object
+  mendota_file <- output_path
+  # fetch data and overwrite file
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
```

Comment:
perhaps here you don't need `mendota_file` and can use `output_path` everywhere that `mendota_file` is used.